src_assets/common/assets/web/configs/tabs/AudioVideo.vue

```diff
@@ -54,12 +54,9 @@ const config = ref(props.config)
         </div>
 
         <!-- Install Steam Audio Drivers -->
-        <div class="mb-3">
-          <label for="install_steam_audio_drivers" class="form-label">{{ $t('config.install_steam_audio_drivers') }}</label>
-          <select id="install_steam_audio_drivers" class="form-select" v-model="config.install_steam_audio_drivers">
-            <option value="disabled">{{ $t('_common.disabled') }}</option>
-            <option value="enabled">{{ $t('_common.enabled_def') }}</option>
-          </select>
+        <div class="mb-3 form-check">
+          <label for="install_steam_audio_drivers" class="form-label">{{ $t('config.install_steam_audio_drivers') }}<div class="mt-0 form-text">{{ $t('_common.enabled_def_cbox') }}</div></label>
+          <input type="checkbox" class="form-check-input" id="install_steam_audio_drivers" v-model="config.install_steam_audio_drivers" true-value="enabled" false-value="disabled" />
```

Comment:
```suggestion
          <input
            type="checkbox"
            class="form-check-input"
            id="install_steam_audio_drivers"
            v-model="config.install_steam_audio_drivers"
            true-value="enabled"
            false-value="disabled"
          />
```

When the line is too long can we do this?

Also, do the true/false values account for all the possible options in the config? Ideally in the cpp we would translate enabled/disabled to boolean before providing it to the web-ui.

Reply:
> When the line is too long can we do this?

I'll try to come up with something better.

> Also, do the true/false values account for all the possible options in the config? Ideally in the cpp we would translate enabled/disabled to boolean before providing it to the web-ui.

As you have said, only 1 value pair will be accounted for. In case none of the values match, it defaults to `false`. Again, this requires more refactoring due to separate parsing in cpp/vue. I do not want to do this here.